src/utils/constants.js

```diff
@@ -1,8 +1,13 @@
 
 // s3 bucket location of 3147 county geoid jsons
-// TODO: fetch should be based on if staging or production
-// export const s3BucketUrl = 'https://idd-dashboard-runs.s3.amazonaws.com/json-files/';
-export const s3BucketUrl = 'https://idd-dashboard-runs-staging.s3.amazonaws.com/json-files/';
+let dataLocation;
+if (process.env.NODE_ENV === 'development') {
+    dataLocation = 'https://idd-dashboard-runs-staging.s3.amazonaws.com/json-files/';
+} else { 
+    // indicate key prefix of relative bucket (could be staging or production)
+    dataLocation = 'json-files/';
+}
+export const s3BucketUrl = dataLocation;
```

Comment:
@BenjiLee turns out react has a very nifty `process.env.NODE_ENV` that tells you if you're in dev or production mode. Yay! Is this syntax messy?

Reply:
Can use this syntax as well
```
const dataLocation = process.env.NODE_ENV === 'development' ?
            'https://idd-dashboard-runs-staging.s3.amazonaws.com/json-files/'
            : 'json-files/';
```

Reply:
I wonder if you can just add the prefix in this code

Reply:
Oh yes, this is much cleaner!